src/python/datastore/data_types.py

```diff
@@ -385,7 +385,7 @@ class Testcase(Model):
   crash_type = StringProperty()
 
   # Crashing address.
-  crash_address = StringProperty(indexed=False)
+  crash_address = TextProperty()
```

Comment:
Will we lose values in old Testcase entities. Many times i have seen this if we change data type, it resets values in old entities ?

Reply:
Nope. TextProperty is equivalent to StringProperty(indexed=False) (but the new NDB requires you to just use TextProperty instead).